scripts/rhel_validation/prepare_lis_vm.sh

```diff
@@ -37,6 +38,15 @@ function prepare_vm {
 	    printf "\n kernel-${kernel_version} installation failed \n"
     fi
     yum -y install kernel-devel-${kernel_version}
+
+    yum -y install yum-utils
+    yumdownloader -y kernel-${kernel_version} kernel-devel-${kernel_version}
+
+    wget https://aka.ms/downloadazcopy-v10-linux
+    tar xvf downloadazcopy-v10-linux
```

Comment:
'downloadazcopy-v10-linux' can be a value in a variable, right?

Reply:
it is a actual file name